docs/site/downloads/cldr-48.md

```diff
@@ -220,33 +250,25 @@ For a full listing, see [Delta Data].
 ## Migration
 
 - Number patterns that did not have a specific numberSystem (such as latn or arab) had be deprecated for many releases, and were finally removed.
-- **TBD — add many items!**
+- The default week numbering changes to ISO instead being based on the calendar week starting in CLDR 48 [CLDR-18275]. The calendar week is will be more clearly targeted at matching usage in displayed month calendars.
+- Additionally, language and territory data in `languageData` and `territoryInfo` data received significant updates to improve accuracy and maintainability [CLDR-18087]
+- The likely language for Belarus changed to Russian [CLDR-14479]
+- The pre-Meiji Japanese eras were removed: There was too much uncertainty in the exact values them 
+and feedback that the general practice for exact dates is to use Gregorian for pre-Meiji dates.
+- **TBD Additional items plus future guidance will be added before the beta, on Oct 1.**
 
 
-### V48 advance warnings
-The following changes are planned for CLDR 48. Please plan accordingly to avoid disruption.
-
-- Any locales that are missing Core data by the end of the CLDR 48 cycle will be removed [CLDR-16004][]
-- The default week numbering will change to ISO instead being based on the calendar week starting in CLDR 48 [CLDR-18275][]. The calendar week data will be more clearly targeted at matching usage in displayed month calendars.
-- The likely language for Belarus is slated to change to Russian [CLDR-14479][]
+### V49 advance warnings
+The following changes are planned for CLDR 49. Please plan accordingly to avoid disruption.
```

Comment:
```suggestion
The following changes are planned for CLDR 49. Please plan accordingly to avoid disruption.
- The default week numbering changes to ISO instead being based on the calendar week starting in CLDR 48 [CLDR-18275]. The calendar week is will be more clearly targeted at matching usage in displayed month calendars.
- The pre-Meiji Japanese eras were removed: There was too much uncertainty in the exact values them 
and feedback that the general practice for exact dates is to use Gregorian for pre-Meiji dates.
```